.github/workflows/cicd-deploy-amplify-branch.yaml

```diff
@@ -143,12 +147,39 @@ jobs:
           !cancelled()
         run: |
           cd terraform && ./deploy.sh ${TERRAFORM_ACTION}
-          webhook_url=$( jq -r '.url.value' amplify-branch_output.json)
+          webhook_url=$( jq -r '.webhook_url.value' amplify-branch_output.json)
           curl -X POST $webhook_url
           echo "app_id=$( jq -r '.app_id.value' amplify-branch_output.json)" >> $GITHUB_OUTPUT
+          echo "app_url=$( jq -r '.app_url.value' amplify-branch_output.json)" >> $GITHUB_OUTPUT
+          echo "app_basic_auth_username=$( jq -r '.app_basic_auth_username.value' amplify-branch_output.json)" >> $GITHUB_OUTPUT
+          echo "app_basic_auth_password=$( jq -r '.app_basic_auth_password.value' amplify-branch_output.json)" >> $GITHUB_OUTPUT
       - name:  Amplify Poll
         if: >-
           always() &&
           !cancelled()
         run: |
           ./scripts/poll_running_amplify_job.sh "${{ steps.terraform_apply.outputs.app_id }}" "${{ needs.metadata.outputs.branch_name }}"
+  accessibility_test:
+    name: "Run accessibility tests"
+    environment: ${{ needs.metadata.outputs.amplify_app_environment }}
+    runs-on: ubuntu-latest
+    needs: [metadata, deploy]
+    timeout-minutes: 10
+    env:
+      BASE_URL: ${{ needs.deploy.outputs.app_url }}
+      BASIC_AUTH_USERNAME: ${{ needs.deploy.outputs.app_basic_auth_username }}
```

Comment:
This leaks the the username and password. 

I've tried utilities like `::add-mask::` but my attempts generally fail. Not sure I fully understand how to use it correctly. 

A suggested approach in-case these output cannot be masked is to write them out into a .env file and have `.pa11y-ci.js` load .env using the `dotenv` npm package.